pkg/services/live/live.go

```diff
@@ -232,9 +240,9 @@ func (g *GrafanaLive) GetChannelHandler(channel string) (models.ChannelHandler,
 	}
 
 	// Parse the identifier ${scope}/${namespace}/${path}
-	id, err := ParseChannelIdentifier(channel)
-	if err != nil {
-		return nil, err
+	id := ParseChannelAddress(channel)
+	if !id.IsValid() {
+		return nil, fmt.Errorf("invalid channel")
```

Comment:
Much better for debugging if the invalid channel is included:
```suggestion
		return nil, fmt.Errorf("invalid channel: %q", channel)
```